### DISCUSSION TITLE
asciidoctor and diagram

### DISCUSSION THREAD

**Discussion by ghost** [manoelcampos/asciidoctor-ghpages-action#22] [Q&A] [answered]
Hello,
Could you help me using the github action I can't generate the diagrams (puml -> svg) they stay in puml format and are not converted, would I have omitted a configuration?

**Comment by manoelcampos** [comment, marked as answer]
I've never used that. The discussion in #16 may help.

**Reply by ghost** [reply]
thanks it is that !!
